guides/release/models/defining-models.md

```diff
@@ -166,14 +166,14 @@ When the API returns a deeply nested, read-only object or array,
 there is no need to create multiple models with `attr('hasMany')` or `attr('belongsTo')`
 relationships. This could result in a potentially large amount of unnecessary
 code. You can access these objects in the template without transforming them. This can be
-done with `attr()` (No attribute type).
+done with `@attr` (No attribute type).
```

Comment:
Wouldn't it be better to write `No transform specified` instead of `No attribute type`? Not part of this PR though.